android/ouinet/src/main/java/ie/equalit/ouinet/ConnectivityBroadcastReceiver.kt

```diff
@@ -0,0 +1,47 @@
+package ie.equalit.ouinet
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import android.net.ConnectivityManager
+import android.net.NetworkInfo
+import android.util.Log
+
+class ConnectivityBroadcastReceiver (
+    background: OuinetBackground
+    ) : BroadcastReceiver() {
+
+    private var background : OuinetBackground? = null
+
+    init {
+        this.background = background
+    }
+
+    override fun onReceive(context: Context?, intent: Intent) {
+        val info = intent.getParcelableExtra<NetworkInfo>(ConnectivityManager.EXTRA_NETWORK_INFO)
+            ?: return
+
+        // Restart the Ouinet client whenever connectivity has changed and become stable.
+        val state = info.state
+        if (state == NetworkInfo.State.CONNECTED || state == NetworkInfo.State.DISCONNECTED) {
+            if (context != null) {
+                //logger.debug("Network state changed to ${state}")
```

Comment:
Is this logger still needed?

Reply:
Nope, I think it was just a holder over from porting this from the Ceno code base, I'll remove it.